source/change-streams/tests/unified/change-streams-showExpandedEvents.yml

```diff
@@ -0,0 +1,299 @@
+      - name: createCollection
+        object: *database0
+        arguments:
+          collection: *existing-collection
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          pipeline: []
+          showExpandedEvents: true
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: insertOne
+        object: *collection0
+        arguments:
+          document:
+            a: 1
+      - name: createIndex
+        object: *collection0
+        arguments:
+          keys:
+            x: 1
+          name: x_1
+      - name: rename
+        object: *collection0
+        arguments:
+          to: *existing-collection
+          dropTarget: true
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult:
+          operationType: insert
+          ns:
+            db: *database0
+            coll: *collection0
+          collectionUUID:
+            $$exists: true
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult:
+          operationType: createIndexes
+          ns:
+            db: *database0
+            coll: *collection0
+          operationDescription:
+            $$exists: true
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult:
+          operationType: rename
+          ns:
+            db: *database0
+            coll: *collection0
+          to:
+            db: *database0
+            coll: *existing-collection
+          operationDescription:
+            dropTarget:
+              $$exists: true
+
+  - description: "when showExpandedEvents is true, createIndex events are reported"
+    operations:
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          pipeline: []
+          showExpandedEvents: true
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: createIndex
+        object: *collection0
+        arguments:
+          keys:
+            x: 1
+          name: x_1
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult:
+          operationType: createIndexes
+
+  - description: "when showExpandedEvents is true, dropIndexes events are reported"
+    operations:
+      - name: createIndex
+        object: *collection0
+        arguments:
+          keys:
+            x: 1
+          name: &index1 x_1
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          pipeline: []
+          showExpandedEvents: true
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: dropIndex
+        object: *collection0
+        arguments:
+          name: *index1
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult:
+          operationType: dropIndexes
+
+  - description: "when showExpandedEvents is true, create events are reported"
```

Comment:
We might want a test for the creation of views, too. Note that we usually use `createCollection` with a `viewOn` argument to do this (see [here](https://github.com/mongodb/specifications/blob/master/source/crud/tests/unified/estimatedDocumentCount.yml#L152-L158)).